Tallly-Core2-V11.ino

```diff
@@ -1151,7 +1162,21 @@ void loop() {
     } else {
       M5.Lcd.fillCircle(ACS_XLOC, ACS_YLOC, ACS_SIZE, TFT_RED);  // ATEM not connected
     }
-  }  // End connection status loop
+  }  // End connection status
+  
+  
+  // Update ATEM "On Air" status
+  if (atemConStatus) {
+    atemOAStatus = AtemSwitcher.getStreamStreaming();
+    if (atemOAStatus != atemLastOAStatus) {
+      atemLastOAStatus = atemOAStatus;
+      M5.Lcd.fillCircle(ACS_XLOC, ACS_YLOC, ACS_SIZE, atemOAStatus ? TFT_RED : TFT_LIGHTGREY);  // ATEM "On Air" Status
```

Comment:
```suggestion
      M5.Lcd.fillCircle(OAS_XLOC, OAS_YLOC, OAS_SIZE, atemOAStatus ? TFT_RED : TFT_LIGHTGREY);  // ATEM "On Air" Status
```

I think you made a copy paste error here. as when you are on air you are changing the color of the connection icon.